utils/random.lua

```diff
@@ -0,0 +1,33 @@
+-- A set of functions to extend math.random and math.randomseed.
```

Comment:
Yes, the `utils` directory is for standalone utility scripts while the root `frontend` directory seems to be a module utils directory.